CWE/Common/Colors.cs

```diff
@@ -1,41 +1,63 @@
-﻿namespace CWE.Common
-{
-    using Discord;
+﻿using System.Diagnostics.CodeAnalysis;
+using Discord;
 
+namespace CWE.Common
+{
     /// <summary>
-    /// Colors used by CWE.
+    ///     Colors used by CWE.
     /// </summary>
-    [System.Diagnostics.CodeAnalysis.SuppressMessage("StyleCop.CSharp.MaintainabilityRules", "SA1401:Fields should be private", Justification = "Fields are used throughout application.")]
+    [SuppressMessage("StyleCop.CSharp.MaintainabilityRules", "SA1401:Fields should be private",
+        Justification = "Fields are used throughout application.")]
     public static class Colors
     {
         /// <summary>
-        /// The color of Discord.
+        ///     The color of Discord.
+        /// </summary>
+        public static Color Discord = new(114, 137, 218);
+
+        /// <summary>
+        ///     The color used to indicate an informative state.
+        /// </summary>
+        public static Color Information = new(26, 155, 226);
+
+        /// <summary>
+        ///     The color used to indicate a success state.
+        /// </summary>
+        public static Color Success = new(95, 218, 153);
+
+        /// <summary>
+        ///     The color used to indicate an error state.
+        /// </summary>
+        public static Color Error = new(236, 56, 69);
+
+        /// <summary>
+        ///     The color used to indicate a warning state.
         /// </summary>
-        public static Color Discord = new Color(114, 137, 218);
+        public static Color Warning = new(254, 184, 6);
 
         /// <summary>
-        /// The color used to indicate an informative state.
+        ///     The color used to indicate an active state.
         /// </summary>
-        public static Color Information = new Color(26, 155, 226);
+        public static Color Active = new(254, 200, 16);
 
         /// <summary>
-        /// The color used to indicate a success state.
+        ///     The color used to indicate a hug.
         /// </summary>
-        public static Color Success = new Color(95, 218, 153);
+        public static Color Hug = new(43, 182, 115);
 
         /// <summary>
-        /// The color used to indicate an error state.
+        ///     The color used to indicate a eight ball response.
         /// </summary>
-        public static Color Error = new Color(236, 56, 69);
+        public static Color eightBall = new(0, 0, 0);
 
         /// <summary>
-        /// The color used to indicate a warning state.
+        ///     The color used to indicate a coinflip.
         /// </summary>
-        public static Color Warning = new Color(254, 184, 6);
+        public static Color Coinflip = new(216, 206, 13);
 
         /// <summary>
-        /// The color used to indicate an active state.
+        ///     The color used to indicate a new community poll.
         /// </summary>
-        public static Color Active = new Color(254, 200, 16);
+        public static Color Poll = new(33, 176, 252);
     }
-}
+}
```

Comment:
This file contains way too many modifications, the only thing that should be different is the addition of i.e. colours for certain commands. Also, as these colours are only used within those commands, there is no need to add them to this class.